tockloader-lib/src/attributes/app_attributes.rs

```diff
@@ -10,12 +10,14 @@ use tbf_parser::{
     types::{TbfFooterV2Credentials, TbfHeader},
 };
 
+// Structure used to package an app's header and footer data.
```

Comment:
A bit barebones for public documentation

```suggestion
/// This structure is used to package together all relevant information about an app. The information is usually stored within a [TbfHeader], and one ore more [TbfFooters](TbfFooterV2Credentials).
///
/// For more details see <link to documentation> or [tbf=parser docs](tbf_parser)
```